setup.py

```diff
@@ -49,6 +49,13 @@ def read(*rnames):
       package_dir={'': 'src'},
       install_requires=[
           'setuptools',
+
+      ],
+      setup_requires=[
+          'pytest-runner',
```

Comment:
It just follows the documentation of pytest: http://doc.pytest.org/en/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner 

In some way, the extras_require test section is also redundant with test_requires